python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -699,5 +712,63 @@ def main():
 
     cleanup()
 
+def verify_credentials(sec_tag, ca_cert, client_cert):
+    write_line('AT%CMNG=1,{},0'.format(sec_tag))
+    retval, ca_res = wait_for_prompt(b'OK', b'ERROR', store=b'\r\n')
```

Comment:
Should it be `store=b'\r\n'` or `store=b'%CMNG:'`?

Reply:
perhaps, i don't remember how that works...